diff --git a/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js b/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
index fe1aa7f..d9a3bb0 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
@@ -321,7 +321,7 @@ export const makeSocket = (config) => {
                 payload: payloadEnc
             }
         }).finish());
-        await noise.finishInit();
+        noise.finishInit();
         startKeepAliveRequest();
     };
     const getAvailablePreKeysOnServer = async () => {
diff --git a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
index 42fb902..caceef1 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
@@ -50,12 +50,10 @@ export const generateLoginNode = (userJid, config) => {
     const { user, device } = jidDecode(userJid);
     const payload = {
         ...getClientPayload(config),
-        passive: true,
+        passive: false,
         pull: true,
         username: +user,
         device: device,
-        // TODO: investigate (hard set as false atm)
-        lidDbMigrated: false
     };
     return proto.ClientPayload.fromObject(payload);
 };
